openapi/rhose-api.yaml

```diff
@@ -936,6 +940,10 @@ components:
           $ref: '#/components/schemas/Source'
         resolvedAction:
           $ref: '#/components/schemas/Action'
+        isErrorHandler:
```

Comment:
This information should not leak to public API. It might be already solved and maybe it is just a matter of update the file